webrender/src/tiling.rs

```diff
@@ -813,6 +817,11 @@ impl ClipBatcher {
                     ClipSource::Image(ref mask) => {
                         let cache_item =
                             resource_cache.get_cached_image(mask.image, ImageRendering::Auto, None);
+                        // Skip the invalid image mask.
+                        if cache_item.texture_id == SourceTexture::Invalid {
```

Comment:
This image mask will be rendered at [1]. So, I think we should skip this image if the texture id is not valid.

[1]
https://github.com/servo/webrender/blob/e11297a240b29bd57154d37ecb402d0b9aa299e6/webrender/src/renderer.rs#L2912